Sources/StreamVideoSwiftUI/CallViewModel.swift

```diff
@@ -140,6 +139,15 @@ open class CallViewModel: ObservableObject {
         didSet {
             if participantsLayout != oldValue {
                 lastLayoutChange = Date()
+
+                let participants = call?.state.participants ?? []
+
+                guard
+                    participantsLayout == .grid,
+                    participants.count < 6,
+                    call?.state.screenSharingSession == nil
+                else { return }
+                participants.forEach { changeTrackVisibility(for: $0, isVisible: true) }
```

Comment:
Why do we need this?

Reply:
I think i can remove that as it was a try to fix the disappearing track but i'm unsure it made any difference. Let me check

Reply:
Removed and provided the fix in #263